xla/service/gpu/gpu_compiler.cc

```diff
@@ -1469,6 +1469,17 @@ absl::Status GpuCompiler::OptimizeHloPostLayoutAssignment(
   pipeline.AddPass<HloCSE>(/*is_layout_sensitive=*/true,
                            /*only_fusion_computations=*/false,
                            /*ignore_control_dependencies=*/true);
+
```

Comment:
Maybe add this only in not-debug builds (wrapped in ```#if NDEBUG```)? For debug builds we would run the verifier before and after each pass, anyway.

Reply:
Good point! Done.